runner/src/main/scala/stress/chain/Folders.scala

```diff
@@ -8,20 +8,28 @@ import stress.config.RunnerConfig
 
 object Folders {
 
+  val RELEASES_FILTER = s"""{"active":false,"planned":true,"completed":false,"onlyMine":false,"onlyFlagged":false,"filter":"","parentId":"Applications/Folder_1"}"""
```

Comment:
Let's replace `filter` with `title` here and possible in other places: AFAIR the filtering name has changed in XLR some releases ago, you can see it in `ReleasesFilter.java`.